### DISCUSSION TITLE
[General] Where is the list of eligible Github people?

### DISCUSSION THREAD

**Discussion by shinji2** [zkSync-Community-Hub/zksync-developers#576] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

The site has a drop for those who have contributed to Github, but where can I find a list of these people?

**Comment by avocadodefi** [comment]
i was about to ask the exact same question, I noticed that my eligibility status shows as zero on the zkSync site for GitHub, despite being listed in the [zkSync Era contributors list](https://github.com/matter-labs/zksync-era/graphs/contributors). Could you please clarify why this discrepancy exists? Thank you!

**Reply by miloserdow** [reply]
I was literally checking top contributors from the key zksync repositories and could not find a single eligible developer

**Reply by avocadodefi** [reply]
yeap, they scammed literally all real developers, if you coded and contributed to zksync repo they excluded you, while they reward professional teams which they pay a monthly salarayx on top. So its like scam, i hope the SEC gonna chase behind them as its like hidden bonus payments, illegal in every country, in germany its a crime, there where zksync is based.

**Comment by miloserdow** [comment]
Has anyone managed to find at least some Github account that is eligible? Maybe their checker is wrong?

**Reply by avocadodefi** [reply]
buddy they scammed all real developers, https://github.com/matter-labs/zksync-era/graphs/contributors exactly 0 of all real develoeprs who distributed to zksync got excluded. You can test and check every single account via zksync checker page. Its a scam, they only pay they own internal devs via github, no real developers, its like "Hidden bonus payments" nothing new or special, its just a crime and illegal ;) Else all normal

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

To comply with GDPR rules, we are not providing a public list of eligible GitHub users. Users can check if their profile is eligible on: https://claim.zknation.io/.

**Reply by avocadodefi** [reply]
yeap, you scammed literally all real developers, anybody  coded and contributed to zksync repo got excluded , while they reward professional teams which they pay a monthly salaray already on top. So its like scam, i hope the SEC gonna chase behind them as its like hidden bonus payments, illegal in every country, in germany its a crime, there where zksync is based.

To see how retard your answer is "regarding GDRP" : https://github.com/matter-labs/zksync-era/graphs/contributors here you can see a list of every zksync contributor, now go and check it via zksync checker... suprise... nobody of them got anything at all. If you wasnt on zksync internals paylist, you wont get anything, simple scam :) 

Welcome to Web3 

feeback was given by one the the top 300 blockchain coders, electric crypto capital was cherish me in their annaul report, so just for credibility :) I also don`t care zksyanc peanuts, i just want to outcall that scam, as a real developer, wo distributed and supported zksync :) <3

**Reply by kadirchan** [reply]
Hi @0xsisyfos I didn't notice I have to submit my wallet for eligible Github users. I was eligible but now it says I am not. Is it possible to extend or add it manually wallet address for GitHub?
